<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 218, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>631</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 191, 188);</string>
    </property>
    <widget class="QWidget" name="mainMenu" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);
</string>
     </property>
     <widget class="QPushButton" name="newSessionButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Start New Session</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewSessionButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>View Session History</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setDateTimeButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Set Date and Time</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="menu">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="play">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>420</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QFrame" name="blueLight">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 255, 0.2);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="greenLight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 255, 0, 0.2);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="redLight">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0, 0, 0.2);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="power">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Power</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="batteryLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>547</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;</string>
     </property>
     <property name="text">
      <string>Battery: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="battery">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>550</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans Mono'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;100%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>450</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upload">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>540</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QWidget" name="sessionMenu" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);
</string>
     </property>
     <widget class="QLabel" name="sessionTimer">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;DejaVu Sans Mono&quot;;</string>
      </property>
      <property name="text">
       <string>0:00</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sessionHistoryMenu" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);
</string>
     </property>
     <widget class="QTextEdit" name="sessionLogs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="setDateTimeMenu" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);
</string>
     </property>
     <widget class="QDateTimeEdit" name="DateTime">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>194</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setDateTime">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="offScreen" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reconnect">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>520</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Reconnect and
Continue</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="eventLog">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>50</y>
      <width>291</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;DejaVu Sans Mono&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Sans Mono&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Event Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Sans Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Reset Neurofeedback</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;DejaVu Sans Mono&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Sans Mono&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PC</string>
    </property>
   </widget>
   <widget class="QFrame" name="adminPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QPushButton" name="lowBattery">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Low Battery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dcHeadset">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Disconnect
Headset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chargeDevice">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Charge Device</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletePCData">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Delete PC Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Sans Mono&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Admin Panel</string>
    </property>
   </widget>
   <widget class="QChartView" name="sinewaves">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>540</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sites">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>500</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
